src/cript/api/utils/save_helper.py

```diff
@@ -32,6 +32,25 @@ def __add__(self, other: "_InternalSaveValues") -> "_InternalSaveValues":
                 return_value.suppress_attributes[uuid] = other.suppress_attributes[uuid]
         return return_value
 
+    def __gt__(self, other):
```

Comment:
this makes me a bit worried overriding the `gt` native method, but not sure if I am wrong or not

Reply:
This is how it's done in python.
Overloading this function allows the use of the `>` operator.

Exactly what we need for the while loop.

And since the code of comparison isn't trivial.
I thought its better to capsule it here.